config/default/kustomization.yaml

```diff
@@ -23,10 +23,10 @@ resources:
 - ../webhook
 - ../certmanager
 
-patchesStrategicMerge:
-- manager_auth_proxy_patch.yaml
-- manager_webhook_patch.yaml
-- webhookcainjection_patch.yaml
+patches:
+- path: manager_webhook_patch.yaml
+- path: webhookcainjection_patch.yaml
```

Comment:
What does `webhookcainjection_patch.yaml` have to do with the operator metrics?

Reply:
Nothing. In this case I commented out the `manager_auth_proxy_patch.yaml` patch and upgraded the `patchesStrategicMerge:` section to `patches:`, as newer kustomize versions warn on `patchesStrategicMerge`

Reply:
:bulb: